pkg/actuators/machine/instances.go

```diff
@@ -373,6 +366,21 @@ func launchInstance(machine *machinev1.Machine, machineProviderConfig *awsprovid
 	return runResult.Instances[0], nil
 }
 
+func buildTagList(machineName string, clusterID string, machineTags []awsproviderv1.TagSpecification) []*ec2.Tag {
+	rawTagList := []*ec2.Tag{}
+	for _, tag := range machineTags {
+		// AWS tags are case sensitive, so we don't need to worry about other casing of "Name"
+		if !strings.HasPrefix(tag.Name, "kubernetes.io/cluster/") && tag.Name != "Name" {
```

Comment:
Do we want to reserve the whole of `kubernetes.io/cluster/*****`? Why not just make sure it's not matching the cluster ID?

Reply:
Yeah, I thought about it, and I think we should.  Because if someone copied a machineset from another cluster and applied it to one in the same account, bad things could happen.